pages/player/PlayerNavBar.tsx

```diff
@@ -0,0 +1,129 @@
+import { useState } from "react";
+import { Player } from "@prisma/client";
+import ScoreBox from "../../components/Player/ScoreBox";
+import TextLayout from "../../components/Player/TextLayout";
+
+interface playerProp {
+  playerProp: Player;
+}
+
+const getContents = (category: string, player: Player): unknown => {
+  switch (category) {
+    case "Overview":
+      return (
+        <div>
+          <div className="mt-12 mb-10 text-2xl font-display">
+            Student Overview
+          </div>
+          <div className="mb-6 text-lg font-bold">Student Bio</div>
+          <div className="m-5">
+            <TextLayout title="About Me" content={player.bio} />
+            <TextLayout title="Hobbies" content={player.bio} />
+            <TextLayout title="Favorite Subject" content={player.bio} />
+            <TextLayout title="Most Difficult Subject" content={player.bio} />
+            <TextLayout title="Siblings" content={player.bio} />
+            <TextLayout title="Parents" content={player.bio} />
+            <TextLayout title="Favorite Fifa Game" content={player.bio} />
```

Comment:
apparently favorite fifa game was a meme so it looks like we don't have to include it anymore :")